src/commcare_cloud/user_utils.py

```diff
@@ -59,28 +60,26 @@ def get_default_ssh_username(host):
     If is_guess is False, the caller should assume the user wants this value
     and should not give them a chance to change their choice of user interactively.
     """
-    for line in subprocess.check_output(['ssh', host, '-G']).decode('utf8').splitlines():
-        if line.startswith("user "):
-            return StringIsGuess(line.split()[1], is_guess=False)
+    ssh_config_file = "~/.ssh/config"
```

Comment:
I don't love this hard-coded path because it feels like the beginning of re-implementing part of `ssh`. Was the switch from ssh subprocess to paramiko strictly necessary? (I also don't love the subprocess, but in this case it feels a little less messy because it uses ssh rather than re-implementing part of it.)

Reply:
I'm not too happy with it myself, but to account only for the most standard path, avoiding added complexity and a series of additional assumptions, seems like a good compromise. We could also parse another environment variable that could be used to specify a different path.

The switch was necessary because `ssh -G` would **always** return a user name, which would be the system user name if one was not matched for that host in the configuration. The advantage of the subprocess is clear, it will correctly parse the entire local ssh configuration regardless of customizations, but unfortunately I wasn't able to find a way to get the value we were thinking we were getting in the original function, which instead would always hit the return inside the conditional statement, rendering it useless for its original purpose.